pkg/kapp/resources/labeled_resources.go

```diff
@@ -13,6 +13,11 @@ import (
 	"k8s.io/apimachinery/pkg/labels"
 )
 
+const (
+	ExistsAnnKey = "kapp.k14s.io/exists" // Value is ignored
+	NoopAnnKey   = "kapp.k14s.io/noop"   // value is ignored
```

Comment:
I was thinking of skipping the check for `noop` annotation as well, but later on just added the question to get thoughts from everyone.
> Question: Should we do it for kapp.k14s.io/noop as well?

Reply:
hmm, i see. i dont have a strong use case in mind for noop being checked as well. may be @100mik got something on his mind?

Reply:
In theory, if a user does apply the noop annotation to a resource associated with another `kapp` application, I think  it will error out in a similar way, since this error occurs while listing existing resources. 

I remember Joe using `--dangerous-override-ownership-of-existing-resources` as a workaround when he was figuring out how package repos with overlapping packages should work. (illustrated in test defined [here](https://github.com/vmware-tanzu/carvel-kapp/pull/483/files#diff-8057c511c4ba289c2097cd0883396a1e1e672d86c719534c414c411de578febdR1))

Reply:
Cool. I will skip the check for resources with noop annotation as well.